backend/src/nodes/utils/image_utils.py

```diff
@@ -138,6 +144,39 @@ def normalize_normals(
     return x, y, z
 
 
+def select_fill(img: np.ndarray, fill: int, for_pil: bool = False):
+    """Select how to fill negative space that results from rotation"""
+
+    c = get_h_w_c(img)[2]
+    if fill == FillColor.AUTO:
+        fill_color = (0,) * c
+    elif fill == FillColor.BLACK:
+        fill_color = (0,) * c if c < 4 else (0, 0, 0, 255 if for_pil else 1)
+    else:
+        img = convert_to_BGRA(img, c)
```

Comment:
This has no effect. `convert_to_BGRA` is a pure function. Right now, `rotate` and `shift` will not convert to RGBA because of this.

Either move this call to `convert_to_BGRA` outside this function or return the adjusted image as well.

Reply:
Oh, whoops. I was only half paying attention when I factored this out